### PR TITLE
Add Kibana Chart

### PR DESCRIPTION
**Pull request by wilkers-steve**
Adds a basic Kibana Chart for use with Elasticsearch and Fluentd

### CODE REVIEW

File: kibana/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: {{ .Values.replicas.kibana }}
+  selector:
+    matchLabels:
+      k8s-app: kibana-logging
+  template:
+    metadata:
+      labels:
```
Comment (larryrensing) [inline kibana/templates/deployment.yaml:40]: define an update strategy in here

File: kibana/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/name: "Kibana"
+spec:
+  ports:
+  - port: {{ .Values.network.port }}
+    protocol: TCP
+    targetPort: {{ .Values.network.target_port }}
+  selector:
+    k8s-app: kibana-logging
```
Comment (larryrensing) [inline kibana/templates/service.yaml]: perhaps add a way for someone to expose a nodeport?

**Review by larryrensing — CHANGES_REQUESTED** [review]

**Comment by alanmeadows** [comment]
Steve, should this be closed as it is now destined for github.com/openstack?

**Comment by wilkers-steve** [comment]
@alanmeadows affirmative

**Pull request closed without merging**